src/styles.less

```diff
@@ -506,10 +506,25 @@ li:hover {
   border-bottom-color: #222;
 }
 
+.selected{
+  color: #080;
+}
+
+.deselected{
```

Comment:
Not sure there needs to be a `deselected` class, that can just be default and then the `selected` class overrides that